api/src/main/java/edu/cornell/mannlib/vitro/webapp/config/ConfigurationPropertiesSetup.java

```diff
@@ -52,6 +52,9 @@
 	/** Configuration property to store the Vitro home directory */
 	private static final String VHD_CONFIGURATION_PROPERTY = "vitro.home";
 
+	/** Configuration property used to determine if there are runtime.properties files in multiple locations **/
+	private static final String RP_MULTIPLE = "rp.multiple";
```

Comment:
Why not give this package-protection (default) instead of private. Then you could refer to it from ConfigurationPropertiesSmokeTests, instead of declaring it a second time.